src/state.c

```diff
@@ -285,6 +288,37 @@ handle_state_mounts_section(GNode* node, struct handler_data* data) {
 	}
 }
 
+static void
+handle_state_devices_section(GNode* node, struct handler_data* data) {
```

Comment:
No check on params - we need every function to check all its parameters to avoid crashes ;)